src/Parlot/Scanner.cs

```diff
@@ -9,13 +9,40 @@ namespace Parlot
     /// </summary>
     public class Scanner
     {
-        public readonly string Buffer;
-        public readonly Cursor Cursor;
+        private static readonly byte MaxWhiteSpacesValue = 162;
+
+        private static readonly byte[] WhiteSpaces = new byte[161] 
```

Comment:
you probably want make this `ReadOnlySpan<byte>`, see https://vcsjones.dev/csharp-readonly-span-bytes-static/